.cirrus.yml

```diff
@@ -116,7 +116,7 @@ task:
         TARGET: x86_64-unknown-linux-musl
         TOOLCHAIN: 1.40.0
   container:
-    image: rust:1.36
+    image: rust:1.40
```

Comment:
Oh crap.  I had a search-and-replace fail on my last PR.  At least this instance had no effect.